src/domain/Reward.ts

```diff
@@ -125,12 +143,12 @@ export function calculateRewards(date: BigInt, pool: Pool): void {
   systemRewards.save()
 }
 
-function checkRewardRate(checker: RewardDayTotal): void {
-  if (checker.toDateRewardAggregateValue.gt(BigDecimal.fromString(tierOneRewards))) {
-    log.debug('pending rewards > 1 MILL:  {}', [checker.toDateRewardAggregateValue.toString()])
+function setRewardRate(systemRewards: RewardDayTotal): RewardDayTotal {
+  if (systemRewards.toDateRewardAggregateValue.lt(BigDecimal.fromString(tierOneRewards))) {
+    log.debug('setting system rewards rate {}', [systemRewards.toDateRewardAggregateValue.toString()])
 
-    // TODO: update this with correct value
-    checker.rewardRate = BigDecimal.fromString('0')
-    checker.save()
+    systemRewards.rewardRate = BigDecimal.fromString('0.0042')
```

Comment:
Can we put the rate here into the config?

Reply:
No this is not a config option and shouldn't be changed ever. Similarly to my previous comment (see https://github.com/centrifuge/tinlake-subgraph/pull/93/files#r550752175) the goal is to keep all reward logic in one file where new reward logic is appended every time we update it.